client/src/pages/settings.jsx

```diff
@@ -24,6 +22,7 @@ export function SettingsPage() {
     return 'An error has occurred retrieving your session configs: ' + error.message;
   }
 
+  // Does xs={12} need to be changed?
```

Comment:
For code cleanliness, it's good to remove any commented-out code from the commit.